components/staticFeeds/sample-widget__switch.json

```diff
@@ -0,0 +1,103 @@
+{
+  "entry": {
+    "canAdd": true,
+    "layoutObject": {
+      "nodeId": "-1",
+      "title": "Switch matching first case",
+      "description": "This widget switches runtime type depending upon a JSON feed.",
+      "url": "https://github.com/uPortal-Project/uportal-app-framework/blob/master/docs/make-a-widget.md#switch-widget-type",
+      "iconUrl": null,
+      "mdIcon": "switch_camera",
+      "fname": "sample-widget__switch",
+      "lifecycleState": "PUBLISHED",
+      "target": "_blank",
+      "widgetURL": "/staticFeeds/enrollment-flag-O.json",
+      "widgetType": "switch",
+      "widgetTemplate": null,
+      "widgetConfig": {
+        "expression": "report[0].enrollmentFlag",
+        "cases": [
+          {
+            "matchValue": "O",
+            "widgetType": "list-of-links",
+            "widgetConfig": {
+              "links": [
+                {
+                  "title": "Matched the first case",
+                  "href": "",
+                  "icon": "fa-at",
+                  "target": "_blank"
+                },
+                {
+                  "title": "Dynamic value was O",
+                  "href": "",
+                  "icon": "fa-at",
+                  "target": "_blank"
+                }
+              ]
+            }
+          }
+        ],
+        "caseotherwise": {
```

Comment:
normally I'd call this the base case, but this is clear enough.

Reply:
Maybe `default` is the typical keyword for this? e.g. [what ng-switch does](https://github.com/uPortal-Project/uportal-app-framework/pull/820/files#diff-893932addcc0222b82433c64708b8c68R67).

Though [JSTL uses c:otherwise](https://www.tutorialspoint.com/jsp/jstl_core_choose_tag.htm), probably what came to mind when I named that.

So, rename to "defaultCase" ?

Reply:
nope it's fine as it is

Reply:
Cool. Posted https://github.com/uPortal-Project/uportal-app-framework/pull/826 as a place to think about the name for this key. I'd be fine with that resulting in a better name, or resulting in confirming that `caseotherwise` is the name to run with.